pytorch_toolkit/formula_recognition/tools/get_config.py

```diff
@@ -0,0 +1,40 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import yaml
+
+
+def get_config(config_path, split):
```

Comment:
I'd rename `split` parameter to `section` or `part` (IMHO `section` is better).

Reply:
Renamed